.eslintrc

```diff
@@ -102,6 +101,30 @@
       "env": {
         "jest": true
       }
+    },
+    {
+      "files": ["src/**", "pages/**", "test/**", "scripts/**"],
+      "rules": {
+        "simple-import-sort/imports": [
+          "warn",
+          {
+            "groups": [
+              // External packages come first.
+              ["^react", "^\\w", "^@testing", "^@vite"],
```

Comment:
What if other external packages are installed? Could we match here any word optionally prefixed with `@`, but excluding `@cloudscape`?

Reply:
good catch! updated